Assets/Chemburn/JumpController.cs

```diff
@@ -0,0 +1,72 @@
+using System;
```

Comment:
Doesn't it make much more sense to place all the new scripts into the Script folder? Maybe divide them in folders: Player, Environment?

Reply:
Yeah will do